encoding/thrift/outbound.go

```diff
@@ -200,6 +164,58 @@ func (c thriftClient) Call(ctx context.Context, reqMeta yarpc.CallReqMeta, reqBo
 	}
 }
 
+func (c thriftClient) buildTransportRequest(
+	reqMeta yarpc.CallReqMeta,
+	reqBody envelope.Enveloper) (
+	transport.Request, protocol.Protocol, error) {
```

Comment:
Return a `*transport.Request` rather than `transport.Request`. `transport.Request` is a heavy object which we shouldn't be copying around too much.